libcontainer/cgroups/fs/memory.go

```diff
@@ -55,38 +67,26 @@ func (s *MemoryGroup) Apply(d *cgroupData) (err error) {
 	return nil
 }
 
-func (s *MemoryGroup) SetKernelMemory(path string, cgroup *configs.Cgroup) error {
```

Comment:
I say again, should we update the systemd manager as well. Right now the systemd manager just does this to "set the kernel memory":

```
func setKernelMemory(c *configs.Cgroup) error {
	path, err := getSubsystemPath(c, "memory")
	if err != nil && !cgroups.IsNotFound(err) {
		return err
	}

	return os.MkdirAll(path, 0755)
}
```

This code **used** to call this function (`apply_raw.go:SetKernelMemory`) directly. It no longer does this, meaning that we're now maintaining two versions of different hacks to enable accounting for cgroups. Can we please fix this (combine the two), or at least verify that it works as intended?